packages/framework/src/hooks/useRegisterBindings.ts

```diff
@@ -23,6 +24,9 @@ export const useRegisterBindings = <T extends Record<string, unknown>>(
 ): RegisterBindingsResult<T> => {
   const { resolvedWidgetId, rootRef } = useWidgetId(id);
   const [widgetState, setWidgetState] = useWidgetState<T>(resolvedWidgetId);
+  const themeContext = useThemeContext(
```

Comment:
The properties can also contain expressions so these need to be merged with the style properties before the expression evaluation